### PR TITLE
make this actually work

### PR DESCRIPTION
**Pull request by KaiSforza**
EleGiggle

### CODE REVIEW

File: hammercloud/hammercloud.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def main():
             return
         if ret:
             subprocess.call('{0}/{1}.sh'.format(hammercloud.cachedir, ret), shell=True)
-    elif isinstance(servers, list):
+    elif isinstance(servers, list) and servers:
```
Comment (gtmanfred) [inline hammercloud/hammercloud.py]: change this, put if not servers above the whole statement, and just return in it, so it skips the above if statement as well

**Comment by KaiSforza** [comment]
Updated the `servers` check.

**Comment by gtmanfred** [comment]
I will tag a release when I get time this afternoon

**Comment by KaiSforza** [comment]
Sounds good. I've got people just using the git repo and `pip install -e` for now.

**Pull request merged**